core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestion.java

```diff
@@ -62,6 +69,23 @@ public CompletionSuggestion(String name, int size) {
         super(name, size);
     }
 
+    public ScoreDoc[] getScoreDocs() {
```

Comment:
can this just be an `Iterable<ScoreDoc>` I think it's only used as a such so we don't need to make a copy of it.